integration

```diff
@@ -311,6 +311,7 @@
   "finity69x2/nws_alerts",
   "firstof9/ha-openei",
   "FL550/dwd_weather",
+  "flexopus/flexopus-hass-sensor",
```

Comment:
PR is not editable.
You need to adjust your branch settings.

Reply:
@ludeeus thanks for the review, updated the branch and added you with write access

Reply:
That is wrong. I will not accept that invitation.
That is not how you make PRs editable if you can not do it.
I suggest you do a new PR not from the "master" branch.
<https://hacs.xyz/docs/publish/include#before-submitting>
> In your fork of hacs/default, create a new branch for your changes from the master branch. Do not use the master branch directly for your changes.

Reply:
thanks for the clarification, and sorry for the confusion! i will get back with a modified PR